src/main/java/utils/WordsProcessingUtils.java

```diff
@@ -0,0 +1,70 @@
+package utils;
+
+import com.google.common.base.Charsets;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import com.google.common.io.CharSource;
+import com.google.common.io.Files;
+import exception.NotEnoughWordsInList;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Set;
+
+public class WordsProcessingUtils {
+    public static String[] findLongestConcatenatedWords(String fileName) throws IOException {
+        List<String> words = createWordsList(fileName);
+        if (words.size() < 4) {
+            throw new NotEnoughWordsInList(
+                    "Please, provide a file with more than three words in it");
+        }
+        List<String> concatenatedWords = createConcatenatedWordsSortedArray(words);
+        if (concatenatedWords.size() < 2) {
+            throw new NotEnoughWordsInList(
+                    "Unfortunately there are not enough concatenated words in your file");
+        }
+        String[] result = new String[3];
+        result[0] = concatenatedWords.get(0);
+        result[1] = concatenatedWords.get(1);
+        result[2] = Integer.toString(concatenatedWords.size());
+        return result;
+    }
+
+    private static List<String> createWordsList(String fileName) throws IOException {
```

Comment:
```suggestion
  List<String> createWordsList(String fileName) throws IOException {
```
Давай приберемо _private_ - тоді ми зможемо покрити цей метод в тестах
Приберемо _static_ - причину вказував вище)